libs/features/seotech/seotech.js

```diff
@@ -7,12 +7,36 @@ export const VIDEO_OBJECT_PROVIDERS = [
   { provider: 'youtube', regex: REGEX_YOUTUBE },
 ];
 
-export function logError(msg) {
-  window.lana?.log(`SEOTECH: ${msg}`, {
+export function logError(msg, context = {}) {
+  // Build additional context string with pilcrow delineation
+  const additionalInfo = [];
+
+  if (context.bucket) {
+    additionalInfo.push(`bucket:${context.bucket}`);
+  }
+
+  if (context.id) {
+    additionalInfo.push(`id:${context.id}`);
+  }
+
+  if (context.videoUrl) {
+    additionalInfo.push(`videoUrl:${context.videoUrl}`);
+  }
+
+  if (context.pathname) {
```

Comment:
What about a shorter version:
```suggestion
['bucket', 'id', 'videoUrl', 'pathname'].forEach((prop) => context[prop] && additionalInfo.push(`${prop}:${context[prop]}`))
```